src/catala/surface/lexer_fr.ml

```diff
@@ -94,7 +94,7 @@ let rec lex_code_fr (lexbuf : lexbuf) : token =
       (* Comments *)
       L.update_acc lexbuf;
       lex_code_fr lexbuf
-  | "*/" ->
+  | "```" ->
```

Comment:
You should move the production rule of `END_CODE` below the one of `BEGIN_CODE`, otherwise all your ``'''catala`` will be parsed as `END_CODE`.

Reply:
I don't think so, indeed, each rule is defined in two distinct functions (`lex_law_fr` and `lex_code_fr`) and they are called according to `L.is_code` so they are no conflicts.